### PR TITLE
Style: Change saved modal styles

### PR DESCRIPTION
**Pull request by bbaa77770**

### CODE REVIEW

**Review by jane-rose — APPROVED** [review]

**Comment by jane-rose** [comment]
Need to fix incorrect path on windows

File: src/app/ui/layouts/AppLayout.jsx [hunk 1]
```diff
@@ -204,9 +205,28 @@ class AppLayout extends PureComponent {
             };
             if (this.props.savedModalType === 'web') {
                 return (
-                    <div className="border-default-black-5 border-radius-4 box-shadow-default width-200">
-                        <div className="sm-flex">
-                            Saved
+                    <div
+                        className={classNames('border-default-black-5', 'border-radius-4', 'box-shadow-module', 'position-ab',
+                            'background-color-white', 'padding-horizontal-10', 'padding-vertical-10', 'bottom-0', 'margin-bottom-16')}
+                        style={{
```
Comment (xiekun1992) [inline src/app/ui/layouts/AppLayout.jsx:211]: define a `class` style

File: src/app/ui/layouts/AppLayout.jsx [hunk 2]
```diff
@@ -226,61 +246,49 @@ class AppLayout extends PureComponent {
...
                 return (
                     <div
-                        className={classNames('border-default-black-5', 'border-radius-4', 'box-shadow-default', 'position-ab',
-                            'background-color-white', 'padding-horizontal-16', 'padding-vertical-16')}
+                        className={classNames('border-default-black-5', 'border-radius-4', 'box-shadow-module', 'position-ab',
+                            'background-color-white', 'padding-horizontal-16', 'padding-vertical-16', 'bottom-0', 'margin-bottom-16')}
                         style={{
                             zIndex: 9999, // TODO?
-                            left: 'calc((100% - 360px) / 2)',
+                            left: '50%',
```
Comment (xiekun1992) [inline src/app/ui/layouts/AppLayout.jsx:253]: define a `class` style

File: src/app/flux/project/index.js [hunk 3]
```diff
@@ -288,14 +289,10 @@ export const actions = {
...
         await UniApi.File.exportAs(targetFile, tmpFile, renderGcodeFileName, (type, filePath = '') => {
-            const pos = filePath.lastIndexOf('/');
-            if (pos > -1) {
-                filePath = filePath.substr(0, pos + 1);
-            }
             dispatch(appGlobalActions.updateSavedModal({
                 showSavedModal: true,
                 savedModalType: type,
-                savedModalFilePath: filePath
+                savedModalFilePath: path.dirname(filePath)
```
Comment (xiekun1992) [inline src/app/flux/project/index.js:295]: refer to `const file = { path: targetFile, name: window.require('path').basename(targetFile) };` in `uni-api.js`, if the path is wrong on Windows

**Review by xiekun1992 — APPROVED** [review]

**Pull request merged**